### PR TITLE
[WIP] feat(drivers): provide a LinstorVolumeManager helper

### PR DESCRIPTION
**Pull request by Wescoeur**

### CODE REVIEW

File: drivers/linstorvolumemanager.py [hunk 1]
```diff
@@ -0,0 +1,1202 @@
...
+                'Could not create volume `{}` from SR `{}`, '.format(
+                    volume_uuid, self._uuid
+                ) + 'resource of the same name already exists in LINSTOR'
+            )
+
+        # I am paranoid.
+        volume_properties = self._get_volume_properties(volume_uuid)
+        if (volume_properties.get(self.PROP_NOT_EXISTS) is not None):
+            raise LinstorVolumeManagerError(
+                'Could not create volume `{}`, '.format(volume_uuid) +
```
Comment (benjamreis) [inline drivers/linstorvolumemanager.py]: This error message seems odd, `Could not get property` seems more accurate IMHO.

File: drivers/linstorvolumemanager.py [hunk 2]
```diff
@@ -0,0 +1,1202 @@
...
+            filter_by_stor_pools=[self._group_name]
+        ).storage_pools
+
+        # TODO: Same idea.
+        capacity = 0
+        for pool in pools:
+            space = pool.free_space
+            if space:
+                capacity += space.free_capacity
+        return capacity * 1024
```
[the remark this answers is not in the document]
Comment (Wescoeur) [reply drivers/linstorvolumemanager.py]: It seems a good idea!

File: drivers/lvhdutil.py [hunk 3]
```diff
@@ -93,7 +93,7 @@ def extractUuid(path):
     return None
 
 def calcSizeLV(sizeVHD):
-    return util.roundup(LVM_SIZE_INCREMENT, sizeVHD)
+    return util.round_up(LVM_SIZE_INCREMENT, sizeVHD)
```
Comment (benjamreis) [inline drivers/lvhdutil.py]: I think you should put the arg in the reverse order now

Comment (Wescoeur) [reply drivers/lvhdutil.py]: Yeah I saw that. ;)

**Pull request merged**